components/ui/themes/PurpleTheme.ts

```diff
@@ -1317,35 +1307,13 @@ export const PurpleTheme = {
   ICON_SMALL_SIZE: 16,
   ICON_MID_SIZE: 22,
   ICON_LARGE_SIZE: 33,
-  PinIcon: require('../../../assets/pin_icon.png'),
   CloseCard: require('../../../assets/card_bg.png'),
-  CardBackground: require('../../../assets/card_bg.png'),
   OpenCard: require('../../../assets/card_bg.png'),
   cardFaceIcon: require('../../../purpleAssets/profile_icon.png'),
```

Comment:
Still Some Images are in png, any specific reason to have this. if not we can still proceed to have svg for this also.

Reply:
As of now there is no way of using the svg image as an background image, since we are using background images for the closed card and open card preview we still require some png images